tutorial/tests/test_input_output.py

```diff
@@ -86,15 +107,18 @@ def test_write_file(function_to_test, tmp_path: pl.Path):
 
 def reference_read_write_file(input_file: pl.Path, output_file: pl.Path) -> None:
     # We open the input and output file at the same time
-    with input_file.open("r") as read_file, output_file.open("w") as write_file:
-        # Here we iterate over each line of the input file
-        for line in read_file.readlines():
-            # We remove line breaks and write the line to the output file
-            clean_line = line.strip("\n\r")
-            # We crete the output line
-            output_line = f"{clean_line}, {len(clean_line)}\n"
-            # Finally we write the line and its length to the output file
-            write_file.write(output_line)
+    read_file = input_file.open("r")
+    write_file = output_file.open("w")
```

Comment:
Why did you remove the context managers? I think it is a good practice to introduce them early.

Reply:
I agree that they are good practice, but it isn't nice to have them in the solutions before properly introduced in the tutorial so here i write it the old fashion way.